source/Planet.cpp

```diff
@@ -231,6 +240,25 @@ void Planet::Load(const DataNode &node)
 
 
 
+// Legacy wormhole do not have an associated Wormhole object so
+// we must auto generate one if we detect such legacy wormhole.
+void Planet::FinishLoading(Set<Wormhole> &wormholes)
+{
+	// If this planet is in multiple systems, then it is a wormhole.
+	if(!wormhole && systems.size() > 1)
+	{
+		wormhole = wormholes.Get(TrueName());
+		wormhole->LoadFromPlanet(this);
+		Files::LogError("Warning: auto generation of wormhole \"" + name + "\".");
```

Comment:
> Let's also print this deprecation for the event case, too.

I mean that's already the case since if a wormhole is created through an event then it goes through the code above already or do you mean something else?